### ISSUE TITLE
RequireNullCoalesceOperator: False positive when aligned with &&

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#427]
Code to reproduce:
```php
$foo = $bar && isset($baz[$bar]) ? $baz[$bar] : null;
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/de61dd250489b0d527b23b11980937c2e56622bb
